types/office-runtime/index.d.ts

```diff
@@ -37,6 +37,15 @@ declare namespace OfficeRuntime {
    * @remarks
    * [Api set: SharedRuntime 1.1, Mailbox 1.10]
    *
+   * This interface is available in the
+   * {@link https://learn.microsoft.com/javascript/api/requirement-sets/common/shared-runtime-requirement-sets | SharedRuntime 1.1 requirement set}
+   * for Excel, PowerPoint, and Word add-ins. It's also available starting in
+   * {@link https://learn.microsoft.com/javascript/api/requirement-sets/outlook/requirement-set-1.10/outlook-requirement-set-1.10 | Mailbox requirement set 1.10}
+   * for Outlook. To learn how to specify a requirement set in your XML manifest, see
+   * {@link https://learn.microsoft.com/office/dev/add-ins/develop/specify-office-hosts-and-api-requirements | Specify Office applications and API requirements}.
```

Comment:
The bit about how to specify a requirement set feels very general, like it could be added to every mention of a requirement set. What's the incentive to add this information here, instead of just leaving it at the first two lines?

Reply:
I was hoping to keep the messaging consistent with other APIs that support multiple requirement sets on different client applications. For example, see [displayDialogAsync](https://learn.microsoft.com/javascript/api/office/office.ui?view=common-js-preview#office-office-ui-displaydialogasync-member(2)).

Reply:
Fair enough for consistency, though maybe we should do a pass to avoid redundancy. It makes sense to me to let the requirement set pages do the heavy lifting, not individual API descriptions.

Reply:
Thanks for the feedback. That makes sense. I removed the redundant lines and added an item to the backlog to do a pass on similar APIs.